doc/manuals/admin/yum.md

```diff
@@ -0,0 +1,49 @@
+## Using Yum repositories
+
+This document describes the guidelines of using FIWARE Yum repository to install Orion Context Broker. Provided configuration corresponds to x86_64 architecture and CentOS/RHEL 7 OS.
+
+There are two available repositories:
+- nightly, for nightly packages.
+- release, for release packages.
+
+You can read about differences between packages [here](install.md##installation) 
```

Comment:
I'm not sure but I'd say that the the anchor uses only one '#'. Please check it.

Reply:
fixed in c5fdc4b640f9d164caf260e8ef3b8947e547c473